libnavigation-router/src/main/java/com/mapbox/navigation/route/offboard/router/ElapsedTime.kt

```diff
@@ -17,7 +16,7 @@ internal class ElapsedTime(
 
     fun end() {
         if (start == null) {
-            throw NavigationException("Must call start() before calling end()")
+            throw RuntimeException("Must call start() before calling end()")
```

Comment:
```suggestion
            throw IllegalStateException("Must call start() before calling end()")
```

Reply:
Removed altogether, as we discussed last time it's not used.